app/assets/javascripts/merchants.js

```diff
@@ -36,10 +38,16 @@ function loadAllMerchants(container) {
   })
 }
 
+let merchantRevenue = function (merchantId) {
```

Comment:
similar to merchantName, this will set the revenue from the promise to merchantRevenue then put it in the view